.github/workflows/ci.yml

```diff
@@ -136,3 +136,8 @@ jobs:
           version: 0.13.8
       - run:
           test "$(tea --version)" = "tea 0.13.8"
+  root:
+      runs-on: ubuntu-latest
+      steps:
+        - uses: actions/checkout@v3
+        - run: sudo su -c './install.sh'
```

Comment:
does this test anything? CI runs as root by default.

Shouldn’t we instead create a new user and then test it runs with `sudo` and then change the permissions so that user can write to `/usr/local/bin` and then uninstall sudo and check it works?